### DISCUSSION TITLE
Yandex image ads

### DISCUSSION THREAD

**Discussion by Jokopentil** [uBlockOrigin/uBOL-home#72] [Q&A]
Can i block ads on Yandex image search
What filter name must activing

**Comment by MasterKia** [comment]
Try the `Optimal` or `Complete` mode in uBOL settings.

What browser are you using? If it's Opera, then you have to allow the extension to work on search pages.

But I guess Yandex ads are covered by `RU: Counters` which is not available in uBOL.
@dimisa-RUAdList

**Reply by Jokopentil** [reply]
Kiwi browser android
Can i block with Russian block list?

**Reply by MasterKia** [reply]
Kiwi browser is **not** supported, see:
https://github.com/uBlockOrigin/uBlock-issues/issues/2791#issuecomment-1699419516

**Reply by Jokopentil** [reply]
is there any other solution?  what about Firefox android, I tried activating Russian list if that helps?

**Reply by Jokopentil** [reply]
If not help i can use block elemen ublock fitur

**Reply by MasterKia** [reply]
Use uBlock Origin on Firefox android.

**Reply by Jokopentil** [reply]
Ok thank you for your information

**Reply by MasterKia** [reply]
Also add this filterlist:
```
https://easylist-downloads.adblockplus.org/cntblock.txt
```
